src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts

```diff
@@ -228,9 +231,9 @@ export class TerminalTaskSystem extends Disposable implements ITaskSystem {
 		private _terminalProfileResolverService: ITerminalProfileResolverService,
 		private _pathService: IPathService,
 		private _viewDescriptorService: IViewDescriptorService,
-		private _logService: ILogService,
 		private _notificationService: INotificationService,
 		instantiationService: IInstantiationService,
+		private _taskLogService: ITaskLogService,
```

Comment:
nit: I've been keeping the prop name as `_loService` to be shorter. It doesn't matter so much inside the class what type of log service it is, just that it's a log service.